### DISCUSSION TITLE
Unable to set component data. Public property [$form]

### DISCUSSION THREAD

**Discussion by AnhAnhPT** [livewire/livewire#8493] [7. Bugs (archived)]
### Livewire version

3.4.12

### Laravel version

v10.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Sory for my English.
I encountered a problem when updating the model using javascript. I have 2 live component on same view but 
when I update data in this component, the other component reports an error
![image](https://github.com/livewire/livewire/assets/129494411/43e96127-82e5-44a7-afb6-22fe2994e26f)
![image](https://github.com/livewire/livewire/assets/129494411/742027f3-def2-482f-83df-055fb288863e)
![image](https://github.com/livewire/livewire/assets/129494411/9fea2b3a-d5b2-4826-9081-525d8bb5ae1d)
Versions before 3.4.11 do not have this error.

All wire:models of the index-page are updated when the wire:model of the action-page is updated. However, the wire:model of the index-page does not exist. The wire:model of the action-page causes the error.

### Repository or Wirebox URL

https://wirebox.app/b/yw970

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment]
@AnhAnhPT can you please make your Wirebox publicly available?

**Reply by AnhTiger** [reply]
[Todo.zip](https://github.com/livewire/livewire/files/15437362/Todo.zip)
Sorry, my application is built on many modules and all modules have the same error, this is one of my modules.
In short, I encountered a problem: when using javascript to update the model of this live1, another live2 on the same page also has that update command, but this component itself does not have a model that needs to be updated, leading to an error.

**Reply by PerryvanderMeer** [reply]
Can you please provide a public Wirebox or a repository with some demo code as requested in the instructions? Not a big fan of downloading and opening unknown zip files.

**Reply by AnhTiger** [reply]
> Can you please provide a public Wirebox or a repository with some demo code as requested in the instructions? Not a big fan of downloading and opening unknown zip files.

https://github.com/AnhTiger/Todo

This is an error that caused my entire website to crash. Hope to hear from you soon

**Reply by AnhAnhPT** [reply]
> Can you please provide a public Wirebox or a repository with some demo code as requested in the instructions? Not a big fan of downloading and opening unknown zip files.

sir ?

**Reply by AnhAnhPT** [reply]
@PerryvanderMeer Is there any solution sir, I'm still stuck with this error?
